### DISCUSSION TITLE
When switch chain to BNB( 56, 97) & Polygon, wagmi is automatically disconnected.

### DISCUSSION THREAD

**Discussion by zouhangwithsweet** [wevm/wagmi#503] [General]
When switch chain to BNB( 56, 97) & Polygon, wagmi is automatically disconnected.

**Comment by zouhangwithsweet** [comment]
https://vite-antd-seed.vercel.app/

**Reply by tmm** [reply]
What wallet are you using? There is a long running bug with MetaMask for this https://github.com/MetaMask/metamask-extension/issues/13375

**Reply by zouhangwithsweet** [reply]
Well, it's true that metamask
